### ISSUE TITLE
Select how many cards are printed

### ISSUE THREAD

**Issue by nbouliol** [bndw/wifi-card#63]
It can be nice to have an input to select how many cards should be printed.

So you dont have to print mutlipe time to get more than 1 card

**Comment by olekstomek** [comment]
It seems you can achieve this in the print dialog where you choose the number of copies.
![image](https://user-images.githubusercontent.com/26818304/125942153-7de0aad2-0fe8-41aa-921f-f8576b0277dc.png)

**Comment by nbouliol** [comment]
Yes indeed but it will use as many sheets of paper as cards

**Comment by olekstomek** [comment]
Ok, so you mean printing e.g. 2 cards with WiFi data on one sheet of paper?

**Comment by nbouliol** [comment]
Yes exactly

**Comment by olekstomek** [comment]
Good idea for saving paper and cost. 👍

**Comment by bndw** [comment]
This would depend on the size of the card and the size of paper it's being printed on. I think in order to expose printing multiple cards per sheet we'd also need more explicit management of card size. I'm not sure the best way to balance simplicity with the desire to control for every aspect of the card, but I'll give it some thought.

**Comment by Adelin1310** [comment]
Hey, maybe if you could get the witdth and height of the print-area at the print button press event and divide the height of an A4 page for example, you could get the actual approximative number of possible qr codes in one A4 page

**Comment by Spetsnaz-Dev** [comment]
@Adelin1310 Yes, it would be a good idea to divide the size of each QR codes equally, like for 2 QR codes, we can give each codes 50% of paper area.

**Comment by Adelin1310** [comment]
@Spetsnaz-Dev Exactly, so if there would be a text box in which the user will enter how many of these qr codes he needs, you could easily find out how many pages would it take so that all of them will be printed. 
**A4 is 21cm x 29.7**, so let's say **x = 29.7/height_of_qr**, x should give you how many would enter on one page and let's say **y is how many copies does the user want**, so **y/x** is basically the answer to how many pages would be required.

**Comment by bndw** [comment]
Circling back here, I'm happy with the current feature set. We haven't had this come up since the initial ask so I'm going to close out.
